draft-tulshibagwale-saag-pushpull-delivery.md

```diff
@@ -74,19 +74,42 @@ normative:
 
 
 --- abstract
-In situations where a transmitter of Security Event Tokens (SETs) to a network peer is also a receiver of SETs from the same peer, it is helpful to have an efficient way of sending and receiving SETs in one HTTP transaction. In many cases, such as when using the OpenID Shared Signals Framework (SSF), the situation where each entity is both a transmitter and receiver is getting increasingly common.
+This specification defines how multiple Security Event Tokens (SETs) can be delivered and received to and by an intended recipeint using HTTP POST over TLS.
 
-Using current mechanisms such as "Push-Based Delivery of Security Event Tokens (SETs) Using HTTP" or "Poll-Based Delivery of Security Event Tokens (SETs) Using HTTP" both require two or more HTTP connections to exchange SETs between peers. This is inefficient due to the latency of setting up each communication. This specification enables bi-directional transmission and reception of multiple SETs in one HTTP connection, and enables them to do so over a single HTTP or WebSocket connection. 
+This specification enabled following two usecases -
+
+* In situations where a transmitter of Security Event Tokens (SETs) to a network peer is also a receiver of SETs from the same peer, it is helpful to have an efficient way of sending and receiving SETs in one HTTP transaction. In many cases, such as when using the OpenID Shared Signals Framework (SSF), the situation where each entity is both a transmitter and receiver is getting increasingly common.
+* In situations where a transmitter of Security Event Tokens (SETs) wants to transmit multiple SETs to the reciever in a single HTTP call.
+
+Using current mechanisms such as "Push-Based Delivery of Security Event Tokens (SETs) Using HTTP" or "Poll-Based Delivery of Security Event Tokens (SETs) Using HTTP" both require two or more HTTP connections to exchange SETs between peers. This is inefficient due to the latency of setting up each communication. This specification enables mutiple events to be transmitted in bi-directional transmission and reception of multiple SETs in one HTTP connection, and enables them to do so over a single HTTP or WebSocket connection.
 --- middle
 
 # Introduction
-Entities that exchange SETs {{RFC8417}} with each other ("Transceivers") can do so efficiently using the protocol defined in this specification. This specification extends the mechanisms described in the DeliveryPush {{RFC8935}} and DeliveryPoll {{RFC8936}} with the following additional mechanisms:
+Entities that exchange SETs {{RFC8417}} with each other ("Transceivers") can do so efficiently using the protocols defined in this specification. This specification extends the mechanisms described in the DeliveryPush {{RFC8935}} and DeliveryPoll {{RFC8936}} with the following additional mechanisms:
```

Comment:
Aren't we just defining one protocol, with an option to use it as a one-way multi-push thing?

Reply:
Somehow, we should add the notion of 1 protocol, 2 methods, or something similar. I will update this as per the suggestion

Reply:
I don't necessarily think of this as two methods. It becomes multi-push if one is just a transmitter, and becomes pushpull if both are transmitters and receivers.